pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -113,6 +115,11 @@ style_min: false
   - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
   - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
 
+  - if ["mc", "dance"].include? hoc_launch
+    - if tutorial[:short_code] == "mchoc"
+      - tutorial[:image] = "/images/tutorials/hoc2018/aquatic2018_new.jpg"
+      - tutorial[:shortdescription] = tutorial[:longdescription] = hoc_s(:hoc2018_tutorial_mchoc_description)
```

Comment:
I'm confused about the double re-assignment on this line. The short description and the long description should be the same?

Reply:
It turns out we only use the long description in the tutorial explorer right now but I thought it prudent to set the short description to the same thing, just in case.